### DISCUSSION TITLE
79d12515-a63c-4dfc-9cbc-ed5e363c724a

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9658] [Rules Discussions]
# 79d12515-a63c-4dfc-9cbc-ed5e363c724a

If you need to do something more than once, then there should be a standard for it. At the heart of our philosophy on creating rules and standards is the idea of consistency.

https://www.ssw.com.au/rules/the-value-of-consistency/

**Comment by PothieuG** [comment]
>What makes a Rule memorable or easy to follow?

That one: https://www.ssw.com.au/rules/rules-to-better-email/

That one is memorable because I use it every day at work but also in my personal life.
It really changed the way I'm sending emails, and I think I will not look back :)

**Comment by tiagov8** [comment]
People usually comment on the clarity of my emails, which I attribute to the SSW Rules I learned and followed over the years - nowadays it's kind of natural.

One rule I often point people to is [Do you use descriptive links?](https://www.ssw.com.au/rules/descriptive-links/). It helps with SEO and content design.

**Comment by brookjeynes-ssw** [comment]
> What makes a Rule memorable or easy to follow?

I find a rule easy to follow when it's straight to the point and tells me what I want to know. Having unnecessary fluff just bloats the page

**Comment by JackDevAU** [comment]
I was originally quite skeptical of the ["Number Tasks"](https://www.ssw.com.au/rules/number-tasks-questions/) rule, but it has definitely been helpful for quickly getting the gist of an email. It’s also great for getting people to actually read emails - when they see a numbered task addressed to them, they’re much more likely to pay attention. 🤣

**Comment by TobyChurches** [comment]
Consistency is everything!!

I love our rules for email consistency - The format makes it super easy to digest and communicate your point

**Comment by SamBWagner** [comment]
I find all the email Rules to be the best as email is generally the worst part of any day. The Rules tend to make that experience better

**Comment by jakebayliss** [comment]
I think all the email formatting rules are great! Although I dont read them much anymore, it has generally made my communication much more consistent

**Comment by Calinator444** [comment]
Personally I think there are two main things that make a rule easy to follow

Personality 👑 - The information we provide on SSW Rules is mostly information that can be found on other documentation sites.  When rules feel like a blog written by a person rather than boring user documentation they stand out. (e.g. this rules has a funny video that Ulysses created so I remember it https://www.ssw.com.au/rules/conduct-a-test-please/)

Relevance - Personally I learn from practice, and if I can't apply a rule to my specific project (i.e. https://www.ssw.com.au/rules/do-you-avoid-combining-different-entities/) I tend to forget what I've read.

**Comment by MarcusIrmscherSSW** [comment]
A rule is memorable and easy to follow when it is clear, concise, actionable, and relevant.

**Comment by ricksu978** [comment]
When those around you demonstrate consistency, it naturally encourages you to follow suit.

The "Email Task" is a perfect example of this:
👉 https://www.ssw.com.au/rules/number-tasks-questions/

**Comment by StefStar** [comment]
Rules are memorable when they are short and immediate can make a difference to whatever I'm working on.

**Comment by isaaclombardssw** [comment]
Love the rules for certain things – especially managing admin (calendars, email practices).
They have the best answer to my technical questions every now and then too.

Wish there was a little summary at the top of some of the longer ones 😅.

**Comment by vladislav-kir** [comment]
While I am not a fan of using Emails as Tasks, I find [Email rules](https://www.ssw.com.au/rules/rules-to-better-email/) to be some of the most beneficial. [Numbering tasks](https://www.ssw.com.au/rules/number-tasks-questions/) and [As per our conversation](https://www.ssw.com.au/rules/as-per-our-conversation-emails/) even found their way into my personal life 😅

**Comment by Lewkans** [comment]
The [email rules](https://www.ssw.com.au/rules/rules-to-better-email/) have made my emails less wordy, more consistent, and cleaned up my inbox substantially

**Comment by kikibianc** [comment]
> Or, ever had a time that a rule saved you time (or from making a mistake)?

https://www.ssw.com.au/rules/risks-of-deploying-on-fridays/
I have seen, in the past week alone, a couple cases of software companies deploying production changes on Fridays and not having the staff to support it on the weekends, and I have an email reminder every Friday with this rule in it....

If SSW Rules were followed by more companies and industries, a lot of headaches would be avoided 😁

**Comment by amankumarrr** [comment]
Consistency is vital! It fosters clarity and structure, especially when guided by Rules. While these can be challenged and adapted over time, consistency remains the anchor that ensures stability amidst change.

**Comment by Ben0189** [comment]
Consistency and standardize. Whenever I forget how to do something, like an email format, I always refer back to the rules.

**Comment by AntPolkanov** [comment]
SSW has a lot of rules! One of my favorite ones is about the email Inbox. It was a moment of revelation to see what it is for!

**Comment by ncn-ssw** [comment]
When working on a team, I have found it useful to be able to refer to certain rules (e.g. ["Do you know how to avoid problems in if-statements?"](https://www.ssw.com.au/rules/how-to-avoid-problems-in-if-statements/)) instead of having to provide detailed arguments separately each time.

**Comment by BenDunkerleySSW** [comment]
I think SSW's best rule is the ChatGPT Cheat sheet - https://www.ssw.com.au/rules/chatgpt-cheat-sheet/ 

I have used this countless times and have referred friends and family to it - it has improved my prompting skills massively.

**Comment by wicksipedia** [comment]
I can appreciate the barber story, I've been to a few barbershops where they offer you a beer/whisky/coffee when you're waiting. I've usually okay and don't need anything although when I turn up and they don't offer anything I feel like I'm missing out!

**Comment by jeoffreyfischer** [comment]
> What is an SSW Rule that is memorable to you?

This rule: https://www.ssw.com.au/rules/how-to-take-feedback-or-criticism/
Reason: that rule made me realise that I rarely used the "Action" - now I use it all the time and it has greatly improved my communication :)

**Comment by tkapa** [comment]
A good rule is one that is concise. 
This rule, for example, is good because the sentiment of the rule is contained in the title. The content could be drastically shortened. However, because you understand what it's about from the title, being consistent, you don't care what the content is, really.

**Reply by jaydenalchin** [reply]
+1 - A good and memorable rule is one that follows https://www.ssw.com.au/rules/concise-writing/

**Comment by chrisschultzssw** [comment]
A great video will always make a rule more memorable - something with personality as well as good content.

**Comment by bahjatalaadelssw** [comment]
>What is an SSW Rule that is memorable to you?
https://www.ssw.com.au/rules/record-a-quick-and-dirty-done-video/

It's a great resource for anyone wanting to build up the skills to record video content and practise their presentation delivery.

**Reply by stephanfakossw** [reply]
There's a great rule on giving good and bad examples. The photo used in the rule makes it very memorable 😀

https://www.ssw.com.au/rules/use-bad-and-good-examples/

**Reply by steven0x51** [reply]
The Email rule are great example of consistency .

**Comment by MattParkerDev** [comment]
I quote the barber story from this rule to more people than I'd like to admit 😅

**Comment by Tanya-Leahy** [comment]
>Or, in general what makes a rule memorable?

Memorable rules are concise and easily applicable to everyday workplace situations. I've noticed a significant improvement in my emails and now find myself using the same principles in my personal emails as well.

**Comment by steven0x51** [comment]
The Email rule are great example of consistency.

**Reply by sethdaily** [reply]
Many things I use in my life now
Some examples: numbering tasks, indentation for quoting, replying 'done', making tasks have clear acceptance criteria, using emojis, not anchoring people

**Comment by ZhangRan8** [comment]
I like the email templates. They are so convenient!

**Comment by ClaraFang-SSW** [comment]
I really appreciate https://my.sugarlearning.com/SSW/items/8291/daily-scrum-emails?term=daily+scrum.

Sending out daily Scrum allows me to keep track of everything I've accomplished since joining SSW. Looking back, I can clearly see my progress as a new starter. Additionally, it helps my induction manager and buddy stay informed about my progress and provide support whenever I encounter challenges.

**Comment by jerryluossw** [comment]
> What makes a Rule memorable or easy to follow?

The rule: https://www.ssw.com.au/rules/rules-to-better-email/ is memorable to me.

Before coming to SSW, I didn't often use email. This rule made me deeply aware of the advantages of email and the correct way to use it.
By the way, this is the first rule I've read completely after joining SSW.

**Comment by ZenoWang1999** [comment]
> What is an SSW Rule that is memorable to you?

https://www.ssw.com.au/rules/explain-deleted-or-modified-appointments/
This Rule about explaining deleted or modified appointments left a strong impression on me. 
When I first sent an appointment, Adam replied to my email with suggested changes and included a link to this Rule, which explains the proper format for updating appointments to maintain a clearer timeline.

**Comment by babakamyljanovssw** [comment]
https://www.ssw.com.au/rules/rules-to-better-email/

Rules to Better Email has helped me to write better emails

**Reply by miichaelsmedley** [reply]
The most memorable rule for me is the 'As per our conversation' rule. In my role, it's crucial to document conversations as thoroughly as possible. I have numerous discussions daily, and some of them are pivotal. Capturing what’s been said not only ensures clarity but can also be instrumental in securing deals.

**Reply by tino-liu** [reply]
https://www.ssw.com.au/rules/rules-to-better-email/

I gotta go with the email rule. I've been using this with clients and never had a complaint on how ideas and information are imparted.

**Comment by Aibono1225** [comment]
This Rule: https://www.ssw.com.au/rules/rules-to-better-email/

The email rules are awesome. I've been using them for my personal email as well and will continue to do so.

**Comment by leehawkinsssw** [comment]
The [Rules to Better Email](https://www.ssw.com.au/rules/rules-to-better-email/) are well structured and cover many different aspects of communicating clearly in what is still one of the most common ways for us to communicate, especially with clients. 

The consistency they bring to SSW communication is highly valuable, both internally and when working with clients.

**Comment by jimmidier** [comment]
> What is an SSW Rule that is memorable to you?

The email rule https://www.ssw.com.au/rules/rules-to-better-email/. It's the user manual of SSW styled emailing and is being used the most.

**Comment by ozair-ashfaque** [comment]
Consistency is everything

**Comment by uly1** [comment]
these ones are my favourite!
https://www.ssw.com.au/rules/rules-to-better-email

**Comment by awaltos** [comment]
Email rules like numbered tasks work well when the recipient knows the expectation which is why it is so efficient in SSW. I have used a form of "as per our conversation" personally and think it's a great way of summarising a convo and clarifying intent.

For the wider community, I think the Rules around tech and architecture are very valuable.

**Comment by joshbermanssw** [comment]
Consistency is great not only individually but as a group - ensures that we all know what we're doing, how we're doing and a clear way to communicate this to others

**Comment by bettybondoc** [comment]
I’ve gotten so much out of using https://www.ssw.com.au/rules/rules-to-better-email/

At first, having 80+ rules for email formatting sounded a bit over the top, but honestly, it’s been a game-changer. Since everyone at work follows the same format, emails are super easy to read and make sense right away. I used to write these massive walls of text, but now, breaking tasks down into numbered points makes everything skimmable and way more user-friendly for whoever’s reading it.

**Comment by danielmackay** [comment]
To me a good rules is short and to the point. A quick intro on the pain, then options (with the best one recommended), and ideally good and bad examples so I can quickly see what to do and what NOT to do.
